.github/workflows/django.yml

```diff
@@ -3,6 +3,14 @@ name: Django CI
 on:
   pull_request:
     branches: [master]
+    paths: 
+      - '.github/workflows/**'
+      - 'patient_management_system/**'
+      - 'patients/**'
+      - 'manage.py'
+      - 'requirements.txt'
+      - '!static/**'
```

Comment:
Check if these ecluded files are also concerned to the action.

Reply:
> Django also provides a mechanism for collecting static files into one place so that they can be served easily. Using the collectstatic command, Django looks for all static files in your apps and collects them wherever you told it to, i.e. the STATIC_ROOT. check the full article [here](https://www.digitalocean.com/community/tutorials/working-with-django-templates-static-files)

The above quote explains that the static files are only concerned when the `collectstatic` command is run and the `templates` folder is not related to the action since the action only run the unit tests defined in the `patients` app which only test the entities defined within the app no use of templates until now.

Reply:
OK!